src/app/dashboard-widgets/analytical-report-widget/analytical-report-widget.component.html

```diff
@@ -14,35 +14,9 @@ <h2 id="spacehome-analytical-report-title" class="card-pf-title">
   <div class="card-pf-body">
     <div *ngIf="buildConfigsCount > 0">
       <div class="fabric8-stack-analysis">
-        <ul id="spacehome-analytical-report-list" class="list-group list-view-pf list-view-pf-view" *ngIf="currentBuild && currentBuild.annotations && currentBuild.annotations['fabric8.io/bayesian.analysisUrl'] && stackAnalysisInformation['recommendations']">
-          <div *ngIf="stackAnalysisInformation['showLoader']" class="recommender-overlay">
-            <div class="spinner spinner-lg"></div>
-          </div>
-          <li class="list-group-item" *ngFor="let recommendation of stackAnalysisInformation['recommendations']">
-            <div class="list-view-pf-main-info">
-              <div class="list-view-pf-body">
-                <div class="list-view-pf-description">
-                  <div class="list-group-item-text">
-                    <span class="pficon pficon-info"></span>
-                    <b>{{recommendation.suggestion}}: {{recommendation.action}} {{recommendation.message}}</b> &nbsp; {{recommendation.subMessage}}
-                  </div>
-                </div>
-              </div>
-            </div>
-          </li>
-          <li class="list-group-item" *ngIf="!stackAnalysisInformation['recommendations'] || stackAnalysisInformation['recommendations'].length === 0">
-            <div class="list-view-pf-main-info">
-              <div class="list-view-pf-body">
-                <div class="list-view-pf-description">
-                  <div class="list-group-item-text">
-                    <span class="pficon pficon-info"></span>
-                    <b>No recommendations</b>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </li>
-        </ul>
+        <div *ngIf="currentBuild && currentBuild.annotations && currentBuild.annotations['fabric8.io/bayesian.analysisUrl']">
+          <stack-report-inshort *ngIf="stackUrl && stackUrl !== ''" [buildNumber]="currentBuild.buildNumber" [appName]="currentBuild.buildConfigName" [repoInfo]="currentBuild && currentBuild.repositoryInformation" [stackUrl]="stackUrl && stackUrl"></stack-report-inshort>
```

Comment:
[stackUrl]="stackUrl && stackUrl" seems unecessary - can you just shorten that to [stackUrl]="stackUrl".  There is a lot of checking already in here for values (including the upfront *ngIf).